calc/token.py

```diff
@@ -6,9 +6,16 @@
 # Saved by the test: When these constants were defined the token strings were
 # inadvertently switched. The ``test_eof_token_at_end_of_line`` test caught the
 # error.
-INTEGER, EOF, PLUS = 'INTEGER', 'EOF', 'PLUS'
+INTEGER, EOF = 'INTEGER', 'EOF'
+PLUS, MINUS, TIMES, DIVIDED_BY = "PLUS", "MINUS", "TIMES", "DIVIDED_BY"
```

Comment:
I asked a facebook question about what people found more clear. The consensus us was that TIMES and DIVIDED_BY were more clear.